src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+
+function sortTable(columnIndex, isNumeric) {
+  const rows = Array.from(table.rows).slice(1, -1);
```

Comment:
Consider checking if the table has more than one row before attempting to sort. If the table is empty or has only one row, sorting is unnecessary.